docs/source/ens.rst

```diff
@@ -54,14 +70,30 @@ Use :code:`setAddress` function for setting address for the domain:
 
   await ensBuilder.setAddress('vlad.ethworks.test', '0x001...03');
 
-Also, it's possible to set an address for domain recursively, if the domain doesn't exist, by specifying the appropriate option:
+.. tabs::
 
-.. code-block:: ts
+  .. group-tab:: Waffle 3.0.0
 
-  await ens.setAddress('vlad.waffle.ethworks.tld', '0x001...03', {recursive: true});
+    It is also possible to set an address for domain recursively, if the domain doesn't exist, by specifying the appropriate option:
 
-Use :code:`setAddressWithReverse` function for setting address for the domain and make this domain reverse. Add recursive option if the domain doesn't exist:
+    .. code-block:: ts
 
-.. code-block:: ts
+      await ens.setAddress('vlad.waffle.ethworks.tld', '0x001...03', {recursive: true});
+
+  .. group-tab:: Waffle 2.5.0
```

Comment:
Instead of tabs, let's just add a note that feature was introduced in Waffle 3.0